scripts/make/qemu.mk

```diff
@@ -29,8 +29,16 @@ qemu_args-aarch64 := \
   -machine virt \
   -kernel $(OUT_BIN)
 
+
 qemu_args-y := -m 2G -smp $(SMP) $(qemu_args-$(ARCH)) \
-  -append ";$(ARGS);$(ENVS)"
+  -append ";$(ARGS);$(ENVS)" \
```

Comment:
`\` at the end of this line is confusing.

Reply:
It was a mistake